dashboard/app/controllers/registrations_controller.rb

```diff
@@ -14,7 +14,31 @@ class RegistrationsController < Devise::RegistrationsController
     :migrate_to_multi_auth, :demigrate_from_multi_auth
   ]
   skip_before_action :verify_authenticity_token, only: [:set_student_information]
-  skip_before_action :clear_sign_up_session_vars, only: [:new, :begin_sign_up, :cancel, :create]
+  skip_before_action :clear_sign_up_session_vars, only: [:new, :begin_sign_up, :begin_creating_user, :cancel, :create]
+
+  NEW_USER_PERMITTED_PARAMS = [
```

Comment:
can we move this list into a private method such as [`upgrade_params`](https://github.com/code-dot-org/code-dot-org/blob/feb25cc904207409f5ee157df004a7afb445c15c/dashboard/app/controllers/registrations_controller.rb#L608)?